typegate/tests/utils/q.ts

```diff
@@ -264,26 +277,54 @@ export class Q {
       defaults["Authorization"] = await this.contextEncoder(context);
     }
 
+    const clean = (headers: Record<string, string>) => {
+      for (const [k, v] of Object.entries(headers)) {
+        if (v === "NULL") {
```

Comment:
when (or why) does that happen?

Reply:
NULL is a placeholder value, it can be anything else. It is set when `withoutHeaders(headers: Array<string>)` is used.